src/depends.py

```diff
@@ -177,6 +183,7 @@ def check_sqlite():
     Simply check sqlite3 module if exist or not with hexversion
     support in python version for specifieed platform.
     """
+    # pylint: disable=deprecated-lambda
```

Comment:
maybe this can be rewritten as a comprehension instead of suppressing lambda warning?